### ISSUE TITLE
Back-port mppnccombine changes to ESM1.5 branch

### ISSUE THREAD

**Issue by aidanheerdegen** [ACCESS-NRI/MOM5#18]
Collation of outputs and restarts from ACCESS-ESM1.5 models is failing with error messages like
```
payu: error: Thread 0 crashed with error code 9.
 Error message:
ERROR: missing at least -m from the input fileset.  Exiting.
```

This is because the `mppnccombine` executable built by spack uses the version of `mppnccombine.c` in the ESM1.5 fork which is 11 years old and does not contain the update to support compression

https://github.com/ACCESS-NRI/MOM5/blob/access-esm1.5/src/postprocessing/mppnccombine/mppnccombine.c

Compression was added later

https://github.com/ACCESS-NRI/MOM5/commit/43d76badd6792fd579bcba0cf31f55bd338bc6e5

There was also a fix to report illegal options

https://github.com/ACCESS-NRI/MOM5/commit/da52a7b436e08d072f237b7a2e1c1f6e504ee038

These changes should be back-ported from `master` to `access-esm1.5`.

**Comment by penguian** [comment]
@aidanheerdegen Would it be correct to just copy [src/postprocessing/mppnccombine/mppnccombine.c](https://github.com/ACCESS-NRI/MOM5/blob/master/src/postprocessing/mppnccombine/mppnccombine.c) to the `access-esm1.5` branch?

**Comment by penguian** [comment]
Comparing the commit histories, it does look like just copying the source file is correct.
https://github.com/ACCESS-NRI/MOM5/commits/master/src/postprocessing/mppnccombine/mppnccombine.c
https://github.com/ACCESS-NRI/MOM5/commits/access-esm1.5/src/postprocessing/mppnccombine/mppnccombine.c

**Comment by aidanheerdegen** [comment]
Closed with #19
